adminSiteServer/app.tsx

```diff
@@ -126,22 +126,25 @@ export class OwidAdminApp {
         // Public preview of a Gdoc document
         app.get("/gdocs/:id/preview", async (req, res) => {
             try {
-                const gdoc = await GdocFactory.load(
-                    req.params.id,
-                    GdocsContentSource.Gdocs
-                )
+                await db.knexReadonlyTransaction(async (knex) => {
```

Comment:
The transaction is commited/rolled back automatically depending on whether the promise the passed function returns resolves or rejects. The connection is left open because the express server is running as a deamon.